apps/api/src/routers/dev.py

```diff
@@ -8,16 +9,21 @@
 
 
 @router.get("/impersonate/{ucinetid}")
-async def impersonate(ucinetid: str) -> RedirectResponse:
+async def impersonate(
+    ucinetid: str, application: Optional[str] = None
+) -> RedirectResponse:
     """Simulate a user identity during local development (does not require https)."""
     user = NativeUser(
         ucinetid=ucinetid,
         display_name="Local Dev",
         email="dev@irvinehacks.com",
         affiliations=[],
     )
+    if application is not None:
+        res = RedirectResponse(f"/portal?application={application}", status_code=303)
+    else:
+        res = RedirectResponse("/portal", status_code=303)
```

Comment:
Suggestion/thought: maintaining this behavior for the dev route seems a bit excessive since this is designed for internal usage, but if it helps to have around, sure.

Reply:
I find it useful for quickly logging into different users who have or haven't submitted an application to test to see if the user login and apply flow is correct.